xla/stream_executor/cuda/cuda_command_buffer.cc

```diff
@@ -597,16 +597,17 @@ absl::StatusOr<size_t> CudaCommandBuffer::GetNodeCount() const {
 }
 
 absl::Status CudaCommandBuffer::PrepareFinalization() {
-  // TODO(b/362769658): Remove this workaround when cuda supports conditionals
-  // with empty graphs.
+#if CUDA_VERSION < 12080
```

Comment:
Shouldn't that be a runtime driver version check?

Something like this: 

```
if (parent_->GetDeviceDescription().driver_version() <
      SemanticVersion{12, 8, 0}) {
// ...
}
```

Reply:
Yes, runtime version check is better, I made a update for that.